CHANGELOG.md

```diff
@@ -54,6 +54,9 @@ This project adheres to [Semantic Versioning](https://semver.org/).
   (#[1665](https://github.com/nix-rust/nix/pull/1665))
 - Implemented `Read` and `Write` for `&PtyMaster`
   (#[1664](https://github.com/nix-rust/nix/pull/1664))
+- Changed `getrlimit` and `setrlimit` to use `rlim_t` directly
```

Comment:
This should go in the "Changed" section, rather than "Added".

If you squash and rebase, this LGTM.

Reply:
Done.